packages/ve-hemi-actions/utils.ts

```diff
@@ -50,9 +79,32 @@ export const validateCreateLockInputs = function ({
     return 'lock duration is too long (maximum 4 years)'
   }
 
-  const isSupportedChain = SupportedChains.includes(chainId)
-  if (!isSupportedChain) {
-    return 'chain is not supported'
+  return undefined
+}
+
+export const validateIncreaseAmountInputs = function ({
+  account,
+  additionalAmount,
+  approvalAdditionalAmount = additionalAmount,
+  chainId,
+  tokenId,
+}: {
+  account: Address
+  additionalAmount: bigint
+  approvalAdditionalAmount?: bigint
+  chainId: number
+  tokenId: bigint
+}) {
+  const commonError = validateCommonInputs({
+    account,
+    amount: additionalAmount,
+    approvalAmount: approvalAdditionalAmount,
+    chainId,
+  })
+  if (commonError) return commonError
```

Comment:
In `eslint-config-bloq` we had a rule enabled to prevent `if () return` in one line and always force using curly brackets. We should keep doing so in TS code.